i18n/en_AU.json

```diff
@@ -84,7 +84,7 @@
   "add_command.autocompleteHint.placeholder": "Example: [Patient Name]",
   "add_command.cancel": "Cancel",
   "add_command.description": "Description",
-  "add_command.description.help": "Describe your incoming webhook.",
```

Comment:
I think this is generated by our translation server, so this shouldn't be updated in this PR